backend/api/views.py

```diff
@@ -0,0 +1,28 @@
+# views.py
+
+from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from django.conf import settings
+import os
+
+from .models import MedicationInfo
+
+
+# Create your views here.
+def test_page(request):
+    return HttpResponse('This is a test page to verify if the \
+                        django setup works')
+
+
+def api_home(request):
+    return render(request, os.path.join(settings.FRONTEND_DIR, 'lib',
```

Comment:
Does dart work in a way in which you're returning the entire backend through a backend response? Or does it work like React in which case you never need Django to return anything other than json?